sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Model-Quality-With-SageMaker-Pipelines-On-schedule.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Please only use present tense (will be used -&gt; are used, since we joined the input and output -&gt; since we join the input and output or since the input and output are joined). Make sure this applies to the rest of the notebook and other notebooks in this PR</p><p> </p><p>Spelling mistake: th -&gt; the</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
done